src/components/Clock.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+type State = {
+  time: string,
+};
+
+type Props = {
+  clockName: string
+};
+
+export class Clock extends React.Component<Props, State> {
+  state: State = {
+    time: new Date().toUTCString().slice(-12, -4),
+  };
+
+  interval = 0;
+
+  componentDidMount() {
+    this.interval = window.setInterval(this.updateTime, 1000);
+  }
+
+  componentWillUnmount() {
+    window.clearInterval(this.interval);
+  }
+
+  updateTime = () => {
+    const newTime = new Date().toUTCString().slice(-12, -4);
```

Comment:
You make similar calculations twice, so you should make a separate helper function for this.